react_frontend/next.config.js

```diff
@@ -7,7 +7,11 @@ module.exports = {
        */
       {
         source: '/api/v1/:path*',
-        destination: `http://localhost:8000/api/v1/:path*`,
+        /**
+         * Use one of the following two destinations during development:
+         */
+        destination: 'https://scrounger.zorroa.com/api/v1/:path*',
+        // destination: 'http://localhost:8000/api/v1/:path*',
```

Comment:
Can the domain be pulled out into a config setting somewhere so it doesn't need to be changed in multiple places?

Reply:
This one here is purely always in dev mode. It's a local proxy to avoid having to spin up an nginx server locally if using local Django, or use remote hosted Django directly.

The other one, the redirect in html one, will most likely be removed altogether and replaced by a splash / settings screen to let the user put in the url of their deployment.

They also don't necessarily need to match since one is for the API and the other one is for the frontend, and one is dev only, while the other is for real.